.cirrus.yml

```diff
@@ -1,133 +1,186 @@
+
+  # Google-cloud VM Images
+  IMAGE_SUFFIX: "c6431352024203264"
+  FEDORA_CACHE_IMAGE_NAME: "fedora-${IMAGE_SUFFIX}"
+  PRIOR_FEDORA_CACHE_IMAGE_NAME: "prior-fedora-${IMAGE_SUFFIX}"
+  UBUNTU_CACHE_IMAGE_NAME: "ubuntu-${IMAGE_SUFFIX}"
+
+  # Container FQIN's
+  FEDORA_CONTAINER_FQIN: "quay.io/libpod/fedora_podman:${IMAGE_SUFFIX}"
+  PRIOR_FEDORA_CONTAINER_FQIN: "quay.io/libpod/prior-fedora_podman:${IMAGE_SUFFIX}"
+  UBUNTU_CONTAINER_FQIN: "quay.io/libpod/ubuntu_podman:${IMAGE_SUFFIX}"
 
 
 gcp_credentials: ENCRYPTED[0c639039cdd3a9a93fac7746ea1bf366d432e5ff3303bf293e64a7ff38dee85fd445f71625fa5626dc438be2b8efe939]
 
 
 # Default VM to use unless set or modified by task
 gce_instance:
-    image_project: "libpod-218412"
-    zone: "us-central1-c"  # Required by Cirrus for the time being
-    cpu: 2
-    memory: "4Gb"
-    disk: 200  # Required for performance reasons
-    image_name: "${FEDORA_CACHE_IMAGE_NAME}"
+  image_project: "libpod-218412"
+  zone: "us-central1-c"  # Required by Cirrus for the time being
+  cpu: 2
+  memory: "4Gb"
+  disk: 200  # Required for performance reasons
+  image_name: "${FEDORA_CACHE_IMAGE_NAME}"
 
 
 # This task is critical.  It updates the "last-used by" timestamp stored
 # in metadata for all VM images.  This mechanism functions in tandem with
 # an out-of-band pruning operation to remove disused VM images.
 meta_task:
-    alias: meta
-    name: "VM img. keepalive"
+  alias: meta
+  name: "VM img. keepalive"
+
+  # see bors.toml
+  skip: $CIRRUS_BRANCH =~ ".*\.tmp"
+
+  container: &smallcontainer
+    image: "quay.io/libpod/imgts:$IMAGE_SUFFIX"
+    cpu: 1
+    memory: 1
+
+  env:
+    IMGNAMES: >-
+      ${FEDORA_CACHE_IMAGE_NAME}
+      ${PRIOR_FEDORA_CACHE_IMAGE_NAME}
+      ${UBUNTU_CACHE_IMAGE_NAME}
+    BUILDID: "${CIRRUS_BUILD_ID}"
+    REPOREF: "${CIRRUS_REPO_NAME}"
+    GCPJSON: ENCRYPTED[e8a53772eff6e86bf6b99107b6e6ee3216e2ca00c36252ae3bd8cb29d9b903ffb2e1a1322ea810ca251b04f833b8f8d9]
+    GCPNAME: ENCRYPTED[fb878daf188d35c2ed356dc777267d99b59863ff3abf0c41199d562fca50ba0668fdb0d87e109c9eaa2a635d2825feed]
+    GCPPROJECT: "libpod-218412"
+
+  clone_script: &noop mkdir -p $CIRRUS_WORKING_DIR
+  script: /usr/local/bin/entrypoint.sh
+
+build_task:
+  name: "Build task"
+  alias: build
+
+  # Only run this on PRs, never during post-merge testing.  This is also required
+  # for proper setting of EPOCH_TEST_COMMIT value, required by validation tools.
+  only_if: $CIRRUS_PR != ""
+
+  # see bors.toml
+  skip: $CIRRUS_BRANCH =~ ".*\.tmp"
+  execution_lock: $CIRRUS_BRANCH
+
+  # Runs within Cirrus's "community cluster"
+  container:
+    image: "$FEDORA_CONTAINER_FQIN"
+    cpu: 4
+    memory: 12
+
+  timeout_in: 20m
+
+  script:
+    - export PATH="$PATH:$GOPATH/bin"
+    - "$SCRIPT_BASE/enable_ssh.sh"
+    - go get github.com/vbatts/git-validation
+    - make
+    - make validate
 
-    # see bors.toml
-    skip: $CIRRUS_BRANCH =~ ".*\.tmp"
+gating_task:
+  name: "Gating test"
+  alias: gating
```

Comment:
Let's just nuke this whole `gating_task`.  I don't think it's necessary.

Reply:
I left gating and nuked build_task since it did not work as I expected.